src/flask_container_scaffold/util.py

```diff
@@ -45,3 +50,30 @@ def _parse_cfg(config):
             settings_dict[key] = config[section][key]
         config_dict.update({section: settings_dict})
     return config_dict
+
+
+def parse_input(logger, obj):
+    """
+    Parses incoming request, returns a serializable object to return
+    to the client in all cases. When there is a failure, the
+    object contains error information.
+    :param Logger logger: Instantiated logger object
+    :param BaseModel obj: An object based ona a pydantic BaseModel to
```

Comment:
question: Did you mean BaseApiModel here? Should we check obj's type and raise an exception if it is not a BaseApiModel?

typo: s/ona/on a/

Reply:
will fix, thanks